bridge/src/side_contract.rs

```diff
@@ -60,47 +60,49 @@ impl<T: Transport> SideContract<T> {
         }
     }
 
-    pub fn call<F: ContractFunction>(&self, f: F) -> AsyncCall<T, F> {
+    pub fn call<F: FunctionOutputDecoder>(&self, payload: Vec<u8>, f: F) -> AsyncCall<T, F> {
```

Comment:
`output_decoder` would be a more descriptive name for the last argument than `f`